File: app/src/main/java/com/bkjcb/rqapplication/check/model/CheckItem.java
package com.bkjcb.rqapplication.check.model;

import com.bkjcb.rqapplication.datebase.ObjectBox;

import java.io.Serializable;

import io.objectbox.Box;
import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Id;

/**
 * Created by DengShuai on 2019/12/23.
 * Description :
 */
@Entity
public class CheckItem implements Serializable {
    @Id(assignable = true)
    public long id;
    public String c_id;
    public String zhandianleixing;
    public String beijiandanweiid;
    public String beijiandanwei;
    public String jianchariqi;
    public String jianchajieguo;
    public String beizhu;
    public String filePath;
    public String checkMan;
    public String year;
    public long systime;
    public int status;
    public int type;//0站点检查 1器具检查


    public CheckItem() {
    }

    public CheckItem(int type) {
        this.type = type;
    }

    public static Box<CheckItem> getBox() {
        return ObjectBox.get().boxFor(CheckItem.class);
    }
}

File: app/src/main/java/com/bkjcb/rqapplication/check/model/CheckItem.kt
package com.bkjcb.rqapplication.check.model

import io.objectbox.annotation.Entity
import io.objectbox.annotation.Id
import java.io.Serializable

/**
 * Created by DengShuai on 2019/12/23.
 * Description :
 */
@Entity
class CheckItem() : Serializable {
    @Id(assignable = true)
    var id: Long = 0
    var c_id: String? = null
    var zhandianleixing: String? = null
    var beijiandanweiid: String? = null
    var beijiandanwei: String? = null
    var jianchariqi: String? = null
    var jianchajieguo: String? = null
    var beizhu: String? = null
    var filePath: String? = null
    var checkMan: String? = null
    var year: String? = null
    var systime: Long = 0
    var status = 0
    var type //0站点检查 1器具检查
            = 0

    constructor(type: Int) : this() {
        this.type = type
    }
}